integration/service/create_test.go

```diff
@@ -256,11 +256,13 @@ func TestCreateServiceConfigFileMode(t *testing.T) {
 	defer apiClient.Close()
 
 	configName := "TestConfig_" + t.Name()
-	configResp, err := apiClient.ConfigCreate(ctx, swarmtypes.ConfigSpec{
-		Annotations: swarmtypes.Annotations{
-			Name: configName,
+	configResp, err := apiClient.ConfigCreate(ctx, client.ConfigCreateOptions{
```

Comment:
nit; while we're updating may as well shorten these var to be more  agnostic and so that we don't end up with (further down) `configResp.Response.ID,` which is a mouthful 😂 ; 

```suggestion
	res, err := apiClient.ConfigCreate(ctx, client.ConfigCreateOptions{
```